e2e/terraform/acls/bootstrap-nomad.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+while true :
+do
+    ROOT_TOKEN=$(nomad acl bootstrap | awk '/Secret ID/{print $4}')
+    if [ ! -z $ROOT_TOKEN ]; then break; fi
+    sleep 5
```

Comment:
Should we convey progress here?  5 seconds (or more) seem like a long time for an operator.

```suggestion
    echo -n '.'
    sleep 5
```

Reply:
Ordinarily I'd say we should definitely include that, but it's running in the context of Terraform, which is already printing progress timing.